.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,7 +3,7 @@ Before submitting a PR make sure the following things have been done:
 - [ ] The commits are consistent with our [contribution guidelines](../blob/master/.github/CONTRIBUTING.md)
 - [ ] You've added tests to cover your change(s)
 - [ ] All tests are passing
-- [ ] You've updated the README (if adding/changing middleware)
+- [ ] You've updated the README and/or the `cider.nrepl` ns (if adding/changing middleware)
```

Comment:
I'm guessing no one will forget to update the ns, otherwise their middleware won't work. :-) But we can mention here re-gening the docs with `lein docs`.

Reply:
Guess so. :D